src/Proyecto26.RestClient/RestClientPromise.cs

```diff
@@ -173,6 +173,33 @@ public static IPromise<T> Post<T>(RequestHelper options)
             return promise;
         }
 
+        /// <summary>
+        /// Load data from the server using a HTTP POST request.
+        /// </summary>
+        /// <returns>Returns a promise for an array of values.</returns>
+        /// <param name="url">A string containing the URL to which the request is sent.</param>
+        /// <param name="body">A plain object that is sent to the server with the request.</param>
+        /// <typeparam name="T">The element type of the array.</typeparam>
+        public static IPromise<T[]> PostArray<T>(string url, object body)
+        {
+            var promise = new Promise<T[]>();
+            PostArray<T>(new RequestHelper { Uri = url, Body = body }, promise.Promisify);
+            return promise;
```

Comment:
@jdnichollsc we could actually replace the three lines of those with a simple `return PostArray(new RequestHelper { Uri = url, Body = body })`.

Reply:
Let me see

Reply:
Fixed, thanks! https://github.com/proyecto26/RestClient/blob/master/src/Proyecto26.RestClient/RestClientPromise.cs#L183